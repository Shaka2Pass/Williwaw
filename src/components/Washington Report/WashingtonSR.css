.WashingtonSR{
  background-color: grey;
  border: black 10px;

  h3 {
    text-align: center;
  }
}