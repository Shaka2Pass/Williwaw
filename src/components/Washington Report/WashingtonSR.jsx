import React from 'react';
import PropTypes from 'prop-types';
import styles from './WashingtonSR.css';


const WashingtonSR = ({ timestamp, localTimestamp, issueTimestamp, fadedRating, solidRating, swell, wind, condition, charts }) => (
  <section className={styles.WashingtonSR}>
    <h1>{timestamp}</h1> 
    <h2>{localTimestamp}</h2>
    <h3>{issueTimestamp}</h3>
    <h3>{fadedRating}</h3>
    <h3>{solidRating}</h3>
    <h3>{swell.absMinBreakingHeight}</h3>
    <h3>{swell.absMaxBreakingHeight}</h3>
    {/* <h3>{wind}</h3>
    <h3>{condition}</h3> 
    <h3>{charts}</h3> */}
  </section>
);

WashingtonSR.propTypes = {
  timestamp: PropTypes.number.isRequired,
  localTimestamp: PropTypes.number.isRequired,
  issueTimestamp:PropTypes.number.isRequired,
  fadedRating:PropTypes.number.isRequired,
  solidRating:PropTypes.number.isRequired,
  // swell:PropTypes.string.isRequired,
  // wind:PropTypes.string.isRequired,
  // condition:PropTypes.string.isRequired,
  // charts:PropTypes.string.isRequired,
};
export default WashingtonSR;
