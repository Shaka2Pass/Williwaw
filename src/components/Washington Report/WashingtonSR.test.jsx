import React from 'react';
import { shallow } from 'enzyme';
import WashingtonSR from './WashingtonSR';

describe('WashingtonSR component', () => {
  it('renders WashingtonSR', () => {
    const wrapper = shallow(<WashingtonSR  
      timestamp="1594285200"
      localTimestamp="1594252800"
      issueTimestamp="1594274400"
      fadedRating="0"
      solidRating="1"
      swell="absMinBreakingHeight: 1.69"
      wind="speed: 7"
      condition="pressure: 1017"
      charts="swell: https://charts-s3.msw.ms/archive/wave/750/16-1594285200-1.gif"/>);
    expect(wrapper).toMatchSnapshot();
  });
});

