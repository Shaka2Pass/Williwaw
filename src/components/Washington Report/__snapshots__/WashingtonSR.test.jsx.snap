// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<section
  className="WashingtonSR"
>
  <h3>
    1594285200
  </h3>
  <h3>
    1594252800
  </h3>
  <h3>
    1594274400
  </h3>
  <h3>
    0
  </h3>
  <h3>
    1
  </h3>
  <h3>
    absMinBreakingHeight: 1.69
  </h3>
  <h3>
    speed: 7
  </h3>
  <h3>
    pressure: 1017
  </h3>
  <h3>
    swell: https://charts-s3.msw.ms/archive/wave/750/16-1594285200-1.gif
  </h3>
</section>
`;

exports[`WashingtonSR component renders WashingtonSR 1`] = `
<section
  className="WashingtonSR"
>
  <h3>
    1594285200
  </h3>
  <h3>
    1594252800
  </h3>
  <h3>
    1594274400
  </h3>
  <h3>
    0
  </h3>
  <h3>
    1
  </h3>
  <h3>
    absMinBreakingHeight: 1.69
  </h3>
  <h3>
    speed: 7
  </h3>
  <h3>
    pressure: 1017
  </h3>
  <h3>
    swell: https://charts-s3.msw.ms/archive/wave/750/16-1594285200-1.gif
  </h3>
</section>
`;
